text/text.go

```diff
@@ -300,3 +302,51 @@ func Draw(dst *ebiten.Image, text string, face font.Face, x, y int, clr color.Co
 
 	textM.Unlock()
 }
+
+// MeasureString measures the size of a given string using a given font.
+//
+// text is the string that's being measured.
+// face is the font for text rendering.
+//
+// Be careful that the passed font face is held by this package and is never released.
+// This is a known issue (#498).
+//
+// MeasureString is concurrent-safe.
+func MeasureString(text string, face font.Face) (int, int) {
+	textM.Lock()
+
+	x, y := 0, 0
+	w, h := 0, 0
```

Comment:
I'd keep `w`, `h` as `fixed.Int26_6`, and convert them to floats at last. What do you think?

Reply:
I intentionally return ints because this method is supposed to return text size in whole pixels. I suppose it might make sense to have these start out as float64s and only cast them to ints when returning, but that *might* lead to rounding errors. I'll investigate.

Reply:
I don't think you have to treat float64 in the calculations. Keeping `fixed.Int26_6` and converting them to ints at the end should be fine.

Reply:
`math.Max` only returns float64s, I'd have to cast that back to `fixed.Int26_6` if I declared `w` and `h` as `fixed.Int26_6`. Functionally, there's no real difference whether the method uses one or the other.

Reply:
Unfortunately, `Round()` would lose some information.

I'd create min/max function for Int26_6.

Reply:
I'll resolve this because all calls to math.Max were removed in b03ea20.